preact.config.js

```diff
@@ -61,4 +67,21 @@ export default (config, env, helpers) => {
       }
     }
   }
+
+  if (!env.production) {
```

Comment:
Would prefer a feature flag for this. We don't want our builds to inject config if say, we accidently forget to set NODE_ENV.

Reply:
[preact-cli docs](https://github.com/preactjs/preact-cli/wiki/Config-Recipes#conditional-changes-based-on-env) seem to suggest using `env.production` to test whether or not the cli was invoked as `dev` or `build`.